### DISCUSSION TITLE
Per-monitor Auto-hide Taskbar settings

### DISCUSSION THREAD

**Discussion by VelocityRa** [valinet/ExplorerPatcher#2524] [Ideas]
My third monitor is an OLED TV, and I would appreciate being able to set Auto-hide only for it (to prevent burn in).

I have not come across any way of doing this besides using [DisplayFusion](https://www.displayfusion.com/) which is kind of performance-intensive, has some lacking features (it has its own taskbar implementation), and is paid software.

**Comment by pyrates999** [comment]
Unless MS has code that does this already, then sure.  Otherwise it's not possible.
